src/webgl/p5.RendererGL.js

```diff
@@ -2391,6 +2391,44 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
 
     return triangleVerts;
   }
+
+  image3D(img,sx,sy,sz,sWidth,sHeight,dx,dy,dWidth,dHeight) {
```

Comment:
This [shares a lot of code with `p5.RendererGL.image`](https://github.com/processing/p5.js/blob/0a7d3d269f4afe7df2bd2e90233175ceeb28db68/src/webgl/3d_primitives.js#L3394), maybe we could make that function support a z coordinate rather than duplicating the code for this case?

Reply:
The code uses `3D primitives`  and changes to `p5.RendererGL `has been removed